runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/state/StateRequestHandlers.java

```diff
@@ -278,6 +291,11 @@ public static StateRequestHandler forSideInputHandlerFactory(
       }
     }
 
+    @Override
+    public Iterable<BeamFnApi.ProcessBundleRequest.CacheToken> getCacheTokens() {
```

Comment:
I don't think we need this since this is the default implementation?

Reply:
That's correct. Thanks.